Sources/EpoxyCore/SwiftUI/LayoutUtilities/SwiftUIMeasurementContainer.swift

```diff
@@ -96,13 +96,24 @@ public final class SwiftUIMeasurementContainer<SwiftUIView, UIViewType: UIView>:
     super.layoutSubviews()
 
     // We need to re-measure the view whenever the size of the bounds changes, as the previous size
-    // will now be incorrect.
+    // may now be incorrect.
     if latestMeasurementBoundsSize != nil, bounds.size != latestMeasurementBoundsSize {
       // This will trigger SwiftUI to re-measure the view.
       invalidateIntrinsicContentSize()
     }
   }
 
+  public override func invalidateIntrinsicContentSize() {
+    super.invalidateIntrinsicContentSize()
+
+    // Invalidate the resolved strategy in case it changes with the re-measurement.
+    _resolvedStrategy = nil
+
+    // Whenever the intrinsic size is invalidated, we need to re-measure to report the new
+    // `measuredFittingSize` to SwiftUI if it's changed.
+    measureView()
```

Comment:
Not re-measuring in `invalidateIntrinsicContentSize` (which we had done previously) had broken our `ExpandableLabel` expansion behavior. Now it works again!

Reply:
Ah good catch! Ideally we could defer the measurement since `invalidateIntrinsicContentSize` can be called quite frequently by the system. Would it be too late to do it somewhere like `layoutSubviews`?